### PR TITLE
Draggable Reorder

### PR DESCRIPTION
**Pull request by aliciakw**
Create grid and list-item wrapper components using the Muuri library to allow users to change the order of fragment cards by clicking and dragging.

**QA Instructions**
In application.js change line 5 to `const DEV = true;`
In your terminal `ember serve`.
Navigate to `localhost:4200/` to see the preview page.

Click and drag cards. Confirm the order is persisted by looking at the JSON preview.
Add a new card and/or edit data. Confirm data is persisted as expected.

### CODE REVIEW

File: app/templates/index.old.hbs [hunk 1]
```diff
@@ -0,0 +1,190 @@
+{{#if extension.data._schema}}
```
Comment (joshiefishbein) [inline app/templates/index.old.hbs]: We still need this file?

Comment (aliciakw) [reply app/templates/index.old.hbs]: nope nope`

**Review by joshiefishbein — APPROVED** [review]

**Comment by hhff** [comment]
This is awesome

Do you think we could set it up to visually behave like the reorderable interaction on contentful proper?

- [x] Has a draggable handle (which has a darkened hover effect)
- [x] Hovers up slightly when hovered anywhere

image

File: app/components/muuri-grid-component.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import Component from '@ember/component';
+import Muuri from 'muuri';
+
+export default Component.extend({
+  classNames: ['muuri-grid-component'],
+  didRender() {
```
Comment (hhff) [inline app/components/muuri-grid-component.js]: Will this this work on `didInsertElement`?

`didRender` is called multiple times:
https://guides.emberjs.com/release/components/the-component-lifecycle/#toc_making-updates-to-the-rendered-dom-with-didrender

**Review by hhff — CHANGES_REQUESTED** [review]

File: app/components/muuri-grid-component.js (hunk 2 above)
Comment (hhff) [reply app/components/muuri-grid-component.js]: We should probably tear this binding down too on willRemoveElement hook

**Review by limsohee1002 — APPROVED** [review]

**Comment by aliciakw** [comment]
screen shot 2019-01-08 at 4 20 32 pm

Updated to look like Contentful @hhff !  @joshiefishbein says Ample is asking for this feature so lmk if it looks good to merge.

**Pull request merged**